ansible/configs/kni-osp/default_vars_osp.yml

```diff
@@ -0,0 +1,5 @@
+---
+ansible_user: cloud-user
+remote_user: cloud-user
+
+chomped_zone_internal_dns: "students.osp.opentlc.com"
```

Comment:
This zone is primarily used for development and testing and is not well maintained and cleaned up. We should discuss what you are using this for and make sure there is not a more appropriate solution.

Reply:
this is the zone that was given during agnosticd training, please provide the one to use

Reply:
`osp_cluster_dns_zone` shoudl be var for this